samples/ios/Continote/Continote/Utils.swift

```diff
@@ -42,12 +44,48 @@ extension MDCRaisedButton {
   }
 }
 
+extension MDCSnackbarManager {
+  /**
+   Shows the provided text in a snackbar message.
+   
+   See: https://material.io/components/ios/catalog/snackbars/
+
+   - Parameter text: The text to show.
+   */
+  static func show(_ text: String) {
+    MDCSnackbarManager.show(MDCSnackbarMessage(text: text))
+  }
+}
+
 extension UILabel {
   /**
    Applies a standardized visual style to this label of a certain kind.
+
+   - Parameter labelKind: The kind of label to appear as.
    */
   func applyAppTheme(for labelKind: Constants.Theme.LabelKind) {
     font = labelKind.getFont()
     alpha = labelKind.getAlpha()
   }
+
+  /**
+   Sets this label's text to the provided value, or the provided placeholder if the value is
+   nil or empty.
+
+   If the text is set to the provided value, the fontIfValueUsed font will be applied to the label.
+   Otherwise, if the placeholder is used, an italicized version of the font will be applied.
+
+   - Parameter value: The value to set the text to.
+   - Parameter placeholder: The placeholder text to use if the value above is nil or empty.
+   - Parameter fontIfValueUsed: The font to use if the text is set to the provided value.
+   */
+  func setText(to value: String?, or placeholder: String, using fontIfValueUsed: UIFont) {
```

Comment:
`desiredFont` may be better here, typically I think it's best practice to avoid conditionals in variable names.